processing/src/main/java/org/apache/druid/math/expr/vector/VectorProcessors.java

```diff
@@ -85,11 +85,14 @@ public static <T> ExprVectorProcessor<T> makeSymmetricalProcessor(
    *
    * @see org.apache.druid.math.expr.ConstantExpr
    */
-  public static <T> ExprVectorProcessor<T> constant(@Nullable String constant, int maxVectorSize)
+  public static <T> ExprVectorProcessor<T> constant(@Nullable Object constant, int maxVectorSize, ExpressionType type)
```

Comment:
why did this change to `Object`? what sorts of non-String objects might be passed in? could you please update the javadoc?

Reply:
oh, i needed a way to make vector constants for basically anything not a numeric primitive, so this method was updated to make all of the object constants, will update javadoc